docs/distributions/ubuntu/faq.md

```diff
@@ -4,11 +4,13 @@ This seems to happen with certain MacBooks because the GRUB bootloader installer
 
 There is a way to stop this. Boot into the installation media with an External USB Keyboard plugged in. Press e when you selected the "Try Ubuntu without installing" option.
 
-Scroll to the bottom with the arrow keys, and type in ``efi=noruntime``. With the External keyboard, then press CTRL+X or F10 to boot into the Live Media. The installation should work fine now.
+Scroll to the bottom with the arrow keys, and type ``efi=noruntime``. With the External keyboard, then press CTRL+X or F10 to boot into the Live Media. The installation should work fine now.
```

Comment:
This part is not required cause the ISOs have efi=noruntime enabled already. I have removed those entries in which efi=noruntime wasn't there.

Reply:
You probably can instead add the fix for blank screen after boot, asking users to boot using rEFInd instead